patroni/__main__.py

```diff
@@ -61,18 +100,41 @@ def load_dynamic_configuration(self) -> None:
                 time.sleep(5)
 
     def get_tags(self) -> Dict[str, Any]:
+        """Get custom tags configured for this node, if any.
+
+        A custom tag is any tag added to the configuration ``tags`` section that is not one of ``clonefrom``,
+        ``nofailover``, ``noloadbalance`` or ``nosync``.
```

Comment:
This is not exactly what code does.
In the config file we could have `nofailover: false`. The code just skips `nofailover` tag in order to not be very verbose because `false` is already a default value.

But, all tags that are not from predefined list are always returned as it is.

Reply:
Thanks for the note, just submitted a commit.